src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,132 @@
+import classNames from 'classnames';
+import React, { ChangeEvent } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../SearchLink';
+import { getSearchWith } from '../../utils/searchHelper';
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get('sex') || null;
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const onQueryChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+    const newSearchParams
+      = getSearchWith(searchParams, { query: value || null });
+
+    setSearchParams(newSearchParams);
+  };
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        <SearchLink
+          className={classNames({
+            'is-active': !sex,
+          })}
+          params={{
+            sex: null,
+          }}
+        >
+          All
+        </SearchLink>
+
+        <SearchLink
+          className={classNames({
+            'is-active': sex === 'm',
+          })}
+          params={{
+            sex: 'm',
+          }}
+        >
+          Male
+        </SearchLink>
+
+        <SearchLink
+          className={classNames({
+            'is-active': sex === 'f',
+          })}
+          params={{
+            sex: 'f',
+          }}
+        >
+          Female
+        </SearchLink>
+      </p>
+
+      <div className="panel-block">
+        <p className="control has-icons-left">
+          <input
+            data-cy="NameFilter"
+            type="search"
+            className="input"
+            placeholder="Search"
+            value={query}
+            onChange={onQueryChange}
+          />
+
+          <span className="icon is-left">
+            <i className="fas fa-search" aria-hidden="true" />
+          </span>
+        </p>
+      </div>
+
+      <div className="panel-block">
+        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+          <div className="level-left">
+            {['16', '17', '18', '19', '20'].map(century => (
```

Comment:
I think it's better to make a variable for this array

Reply:
Agree, thank you!